src/Sylius/Bundle/AdminBundle/Resources/config/app/events.yaml

```diff
@@ -613,19 +613,19 @@ sylius_ui:
                     priority: 45
                     context:
                         event: sylius.admin.order.show.before_addresses
-                addresses:
-                    template: "@SyliusAdmin/Order/Show/_addresses.html.twig"
-                    priority: 40
                 before_payments_legacy:
                     template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
-                    priority: 35
+                    priority: 40
                     context:
                         event: sylius.admin.order.show.before_payments
                 payments:
                     template: "@SyliusAdmin/Order/Show/_payments.html.twig"
-                    priority: 30
+                    priority: 35
                 shipments:
                     template: "@SyliusAdmin/Order/Show/_shipments.html.twig"
+                    priority: 30
+                addresses:
```

Comment:
Since the `addresses` node has been moved, shouldn't `before_addresses_legacy` follow suit?

Reply:
Done :D